schema/fmi3BuildDescription.xsd

```diff
@@ -108,7 +108,7 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             <xs:attribute name="fmiVersion" use="required">
                 <xs:simpleType>
                     <xs:restriction base="xs:normalizedString">
-                        <xs:pattern value="3[.](0-.*)"/>
+                        <xs:pattern value="3[.](.*)"/>
```

Comment:
```suggestion
                        <xs:pattern value="3[.](0|[1-9][0-9]*)([.](0|[1-9][0-9]*))?(-.+)?"/>
```

Reply:
@andreas-junghanns This should cover the parts of semantic versioning we care about, while still allowing just "3.0", without any build-variants, which are not applicable... Still have to test this somewhat, though...

Reply:
Shouldn't e.g. `3.0.1-beta.1` also be a valid version?

Reply:
It is, or do I miss something?

Reply:
> It is, or do I miss something?

We also allow now `3.0-gamma42Beta` - right?